app/views/admin/users/index.html.erb

```diff
@@ -135,6 +135,7 @@
 </div>
 <% if FeatureFlag.enabled?(:member_index_view) %>
  <%= render "admin/users/index/filters_modal" %>
- <%= render "admin/users/modals/add_organization_modal" %>
  <%= render "admin/users/modals/add_role_modal" %>
+ <%= render "admin/users/modals/add_organization_modal" %>
```

Comment:
I moved this below the add role modal partial since the order in which we have the buttons, tests, etc. are in the order of "Add role" first and "Add organization" second. _I know, I know,_ this is super nitpicky, but I really like to see it all consistently ordered, regardless of the file. 😂

Reply:
This kinda makes me wanna alphabetise them all 🤓

Reply:
Don't tempt me, because I probably will! 😂